lib/proxi/local/sdrf_handler.py

```
@@ -0,0 +1,729 @@
+            if icolumn is None:
+                self.log_problem('WARNING', 'MissingRecommendedColumn', 0, 0, f"Column {title} is missing")
+
+
+
+    ####################################################################################################
+    #### Check and potentially fix the header values
+    def compute_stats(self):
+
+        sdrf_data = self.sdrf_data
+        if sdrf_data is None:
+            eprint(f"ERROR: SDRFData.check_and_fix_headers: No available data to process")
```

Ruff (0.15.12)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)